### PR TITLE
Added negative coloring for currency input

### PR DESCRIPTION
**Pull request by eversteeg**
### Pull Request (PR) Dexels-ui-kit

**Jira link**:
*https://jira.sportlink.nl/browse/CW-1167*

**Description of the pull request**:
*- optional negative color for currency input*
*- the color is also adjustable, but has a sane default*

<br />

- [ ] New component? Did you add it to the library exports file `src/lib/index.ts`?
- [ ] New iconfont? Update `selection.json`, `iconfont.css` (mind the path part) and the values in `src/app/types.ts`?

<br />

### CODE REVIEW

File: src/app/components/molecules/Input/Input.tsx [hunk 1]
```diff
@@ -189,6 +194,8 @@ export const Input: FunctionComponent<InputProps & { [key: string]: any }> = ({
         setIsHovered(!isHovered);
     }, [isHovered]);
 
+    console.log('hasNegativeAmountColor', hasNegativeAmountColor, inputValue, toNumber(inputValue) < 0);
```
Comment (ayalalevinger) [inline src/app/components/molecules/Input/Input.tsx:197]: leftover logging in component

File: src/app/styles/constants.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export const COLOR_NEGATIVE_AMOUNT = 'red';
```
Comment (ayalalevinger) [inline src/app/styles/constants.ts:1]: why is this red and not the color coming from the theme?
which is a different kind of red
https://projects.invisionapp.com/d/main#/console/18925362/394271989/preview

File: src/app/components/molecules/Input/Input.tsx [hunk 3]
```diff
@@ -31,8 +32,10 @@ export interface InputProps {
     autoFocus?: boolean;
     children?: never;
     className?: string;
+    colorNegativeAmount?: string;
```
Comment (ayalalevinger) [inline src/app/components/molecules/Input/Input.tsx:35]: why do we need it? can it be a different color other then the color in the theme?

File: src/app/components/molecules/Input/Input.sc.ts [hunk 4]
```diff
@@ -90,6 +92,12 @@ export const TextField = styled.input<TextFieldProps>`
     width: 100%;
     color: ${({ theme }): string => theme.colorTextBody.primary};
 
+    ${({ colorNegativeAmount, hasNegativeAmountColor }): SimpleInterpolation =>
+        hasNegativeAmountColor &&
+        css`
+            color: ${colorNegativeAmount};
```
Comment (ayalalevinger) [inline src/app/components/molecules/Input/Input.sc.ts:98]: why is this a variable and not coming from the theme?

File: src/app/components/molecules/Input/Input.tsx (hunk 1 above)
Comment (eversteeg) [reply src/app/components/molecules/Input/Input.tsx]: Aarrgghh

File: src/app/components/molecules/Input/Input.tsx (hunk 3 above)
Comment (eversteeg) [reply src/app/components/molecules/Input/Input.tsx]: After discussion with the designer, decided to not have an optional color

File: src/app/components/molecules/Input/Input.sc.ts (hunk 4 above)
Comment (eversteeg) [reply src/app/components/molecules/Input/Input.sc.ts]: See previous discussion

**Pull request closed without merging**